fastlane/README.md

```diff
@@ -15,6 +15,15 @@ Install _fastlane_ using
 or alternatively using `brew cask install fastlane`
 
 # Available Actions
+## iOS
+### ios beta
+```
+fastlane ios beta
+```
+Ship to testflight
+
+----
+
 ## Android
```

Comment:
This file has not been updated, but is supposed to be autogenerated whenever fastlane is run